docs/docs/getting-started/first-capsule.mdx

```diff
@@ -109,3 +109,51 @@ rig deploy nginx-capsule -i nginx -r 3 --set-network-interface network.yaml --se
 ```
 
 Which will create a new capsule, add the image, scale it to 3 replicas, set the network interface and mount the config file.
+
+## Declarative Deployment
+
+Since a Capsule can be described declaratively, it is, similarly to `kubectl apply`,
+possible to deploy a capsule using a yaml file describing it.
+
+For example, we can create a `nginx-capsule.yaml` file that describes the capsule in the example above.
+Notice the project and environment fields, which are used to identify and place the capsule in Rig.
+
+```yaml title="nginx-capsule.yaml"
+apiVersion: platform.rig.dev/v1
+kind: Capsule
+environment: prod
```

Comment:
It's missing a name, ala `name: my-capsule`

Reply:
Ye, but you give a name in the deploy command which takes precedence